pandas/computation/__init__.py

```diff
@@ -7,22 +7,13 @@
 try:
     import numexpr as ne
     ver = ne.__version__
-    _NUMEXPR_INSTALLED = ver >= LooseVersion('2.1')
+    _NUMEXPR_INSTALLED = ver >= LooseVersion('2.4.6')
 
-    # we specifically disallow 2.4.4 as
-    # has some hard-to-diagnose bugs
-    if ver == LooseVersion('2.4.4'):
-        _NUMEXPR_INSTALLED = False
-        warnings.warn(
```

Comment:
and fixed a typo in https://github.com/pandas-dev/pandas/pull/15383/commits/7a275ce974fbd85cbbf65f2feda32b06ea836917